pol-core/clib/esignal.cpp

```diff
@@ -15,7 +15,24 @@ namespace Pol
 {
 namespace Clib
 {
-std::atomic<bool> exit_signalled( false );
+std::atomic<bool> signal( false );
+std::atomic<int> code = 0;
+
+void signal_exit( int exit_code )
+{
+  code = exit_code, signal = true;
+}
+
+bool exit_signalled()
+{
+  return signal;
+}
+
+int exit_code()
+{
+  return code;
+}
```

Comment:
Suggest reading these directly rather than having trivial accessors (that then need to have slightly different names)
```suggestion
```

Reply:
Addressed by removing the getter wrappers.